### PR TITLE
Fixing master

### PR DESCRIPTION
**Pull request by marcinel123**
Hi Adam,

I hope I made all changes you told me to do. All files has been checked by pritter.

Honestly I do have a problem with eslint. I installed it and I do have seperate files for it but when I run npx eslint App.js in command line it shows that:
Oops! Something went wrong! :(
ESLint: 8.31.0
No files matching the pattern "App.js" were found.
Please check for typing mistakes in the pattern.

Thanks,
Martin

### CODE REVIEW

File: app/src/App.styles.js [hunk 1]
```diff
@@ -0,0 +1,8 @@
+import styled from "styled-components";
+
+export const StyledWrapper = styled.div`
```
Comment (AdamKniec) [inline app/src/App.styles.js:3]: Formatting looks great! Good job!

File: app/src/Theme.js [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import Blank from "./";
+
+export const theme = {
```
Comment (AdamKniec) [inline app/src/Theme.js:3]: 👏 👏 👏

File: app/src/components/Header/Header.styles.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
...
+export const StyledLogoNavLink = styled(NavLink)`
+  font-size: 28px;
+  color: black;
+  font-weight: 700;
+  text-decoration: none;
+`;
+export const StyledNavLink = styled(NavLink)`
+  margin-left: 30px;
+  font-size: 14px;
+  color: hsl(0, 0%, 75%);
```
Comment (AdamKniec) [inline app/src/components/Header/Header.styles.js]: this should be taken from the theme colors

File: app/src/components/Header/Header.styles.js [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+
+export const StyledButtonsWrapper = styled.div`
+  width: 300px;
+  display: flex;
+`;
+
+export const StyledLoginButton = styled.button`
+  border: none;
+  background-color: white;
+  color: hsl(0, 0%, 75%);
```
Comment (AdamKniec) [inline app/src/components/Header/Header.styles.js]: this should be taken from the theme colors

File: app/src/components/Header/Header.styles.js [hunk 5]
```diff
@@ -0,0 +1,57 @@
...
+`;
+
+export const StyledButtonsWrapper = styled.div`
+  width: 300px;
+  display: flex;
+`;
+
+export const StyledLoginButton = styled.button`
+  border: none;
+  background-color: white;
```
Comment (AdamKniec) [inline app/src/components/Header/Header.styles.js]: this should be taken from the theme colors

File: app/src/components/Header/Header.styles.js [hunk 6]
```diff
@@ -0,0 +1,57 @@
...
+  color: hsl(0, 0%, 75%);
+  font-size: 14px;
+  font-weight: 700;
+`;
+export const StyledSignUpButton = styled.button`
+  border: none;
+  background-color: hsl(180, 66%, 49%);
```
Comment (AdamKniec) [inline app/src/components/Header/Header.styles.js]: Decide on one color type pattern (hex / hsl) and stick to it. Aditionally, move it to theme colors

File: app/src/components/InputElement/InputElement.js [hunk 7]
```diff
@@ -0,0 +1,12 @@
+import React from "react";
+import { StyledForm } from "./InputElement.styles";
+
+export const InputElement = () => {
+  return (
+    <StyledForm>
```
Comment (AdamKniec) [inline app/src/components/InputElement/InputElement.js:6]: Nice!

File: app/src/components/InputElement/InputElement.styles.js [hunk 8]
```diff
@@ -0,0 +1,14 @@
+import styled from "styled-components";
+import background from "../../images/bg-shorten-desktop.svg";
+
+export const StyledForm = styled.form`
+  position: absolute;
+  left: 270px;
+  border: 1px solid black;
```
Comment (AdamKniec) [inline app/src/components/InputElement/InputElement.styles.js]: "black" should be taken from the theme colors

File: app/src/App.js [hunk 9]
```diff
@@ -1,25 +1,23 @@
...
 export const App = () => {
   return (
     <>
-      <GlobalStyles />
-      <Wrapper>
-        <Header />
-        <Content/>
-        <InputElement/>
-      </Wrapper>
+      <ThemeProvider theme={theme}>
```
Comment (AdamKniec) [inline app/src/App.js:13]: This is exactly how to do this. Good job!

Comment (marcinel123) [reply app/src/App.js:13]: thanks :)

File: app/src/components/InputElement/InputElement.styles.js (hunk 8 above)
Comment (marcinel123) [reply app/src/components/InputElement/InputElement.styles.js]: already fixed

File: app/src/components/InputElement/InputElement.js (hunk 7 above)
Comment (marcinel123) [reply app/src/components/InputElement/InputElement.js:6]: thanks

File: app/src/components/Header/Header.styles.js (hunk 6 above)
Comment (marcinel123) [reply app/src/components/Header/Header.styles.js]: already fixed

File: app/src/components/Header/Header.styles.js (hunk 5 above)
Comment (marcinel123) [reply app/src/components/Header/Header.styles.js]: already fixed

File: app/src/components/Header/Header.styles.js (hunk 4 above)
Comment (marcinel123) [reply app/src/components/Header/Header.styles.js]: already fixed

File: app/src/components/Header/Header.styles.js (hunk 3 above)
Comment (marcinel123) [reply app/src/components/Header/Header.styles.js]: already fixed

File: app/src/Theme.js (hunk 2 above)
Comment (marcinel123) [reply app/src/Theme.js:3]: thanks

File: app/src/App.styles.js (hunk 1 above)
Comment (marcinel123) [reply app/src/App.styles.js:3]: thanks

**Comment by marcinel123** [comment]
already made all changes and I commited it.

**Review by AdamKniec — APPROVED** [review]
🎖️

**Pull request merged**